app/views/govuk_component/docs/search.yml

```diff
@@ -0,0 +1,19 @@
+name: "Search"
+description: "Search box"
```

Comment:
Had left the search input name as it is because I assumed the backend was expecting that.